addons/finalAlert/finalAlert.lua

```diff
@@ -0,0 +1,303 @@
+Copyright © 2022, Godchain
```

Comment:
The entire copyright notice is not wrapped in comments. This way, the addon won't load. Wrap it in `--[[` and `]]`.